docs/en/connector-v2/source/Elasticsearch.md

```diff
@@ -19,24 +19,25 @@ support version >= 2.x and <= 8.x.
 
 ## Options
 
-|          name           |  type   | required |   default value   |
```

Comment:
Why is the `index_list` not added?

Reply:
![image](https://github.com/user-attachments/assets/e799b809-6eff-41af-be25-a469967242d3)
after scroll_size

Reply:
ok